src/components/navbar/Menu.tsx

```diff
@@ -26,12 +26,14 @@ import { FiUser, FiUsers } from "react-icons/fi";
 import { HiOutlineBookOpen, HiOutlineSwitchHorizontal } from "react-icons/hi";
 import MenuNav from "./MenuNav";
 import AdminMenu from "./AdminMenu";
+import { useHasPermission } from "@/hooks/useHasPermissions";
 
 const Menu = () => {
   const { data: userSession } = useSession();
-
-  const isAdmin = userSession?.user?.permissions === "admin";
-
+  // Permissions check
+  const isPermitted = useHasPermission("accessAdminNav");
```

Comment:
to follow the same naming convention as the rest of the permissions
```suggestion
  const canAcessAdminNav = useHasPermission("accessAdminNav");
```